test/test-tcp-close-accept.c

```diff
@@ -84,17 +84,17 @@ static void read_cb(uv_stream_t* stream, ssize_t nread, const uv_buf_t* buf) {
   uv_loop_t* loop;
   unsigned int i;
 
-  /* Only first stream should receive read events */
-  ASSERT(stream == (uv_stream_t*) &tcp_incoming[0]);
+  ASSERT((stream == (uv_stream_t*) &tcp_incoming[0]) ||
+         (stream == (uv_stream_t*) &tcp_incoming[1]));
   ASSERT(0 == uv_read_stop(stream));
   ASSERT(1 == nread);
 
   loop = stream->loop;
   read_cb_called++;
 
-  /* Close all active incomings, except current one */
-  for (i = 1; i < got_connections; i++)
-    uv_close((uv_handle_t*) &tcp_incoming[i], close_cb);
+  /* Close the other active connection, not current one */
+  i = (stream == (uv_stream_t*) &tcp_incoming[0]) ? 1 : 0;
```

Comment:
You don't need the ternary, `x == y` always evaluates to either 0 or 1.  You could also write it as:
```c
i = (uv_tcp_t*) stream - &tcp_incoming[0];
```
That will keep on working when more entries are added to the `tcp_incoming` array.  You could do a similar thing for the ASSERT a few lines up.  Anyway, I'll leave it up to you if you want to change it.